### DISCUSSION TITLE
Broken Patch

### DISCUSSION THREAD

**Discussion by akerb00m** [cables-gl/cables_docs#507] [! Welcome - readme first!]
Greetings
I think i actually broke a Patch yesterday, It crashed at some point. Normally i just delete the nodes that caused the crash and the patch would work again but not this time.

Here the patch, let me know if i should invite (collab with) someone to take a look under the hood. https://cables.gl/p/WYrvas

I was testing out the Ammo world, the patch has a 'view' switch to toggle between orbit controls and fps camera, i think the crash happened around the time i added a spotlight to the patch.

Cheers
GJ

**Comment by steam0r** [comment]
which errormessage do you get? please invite me (stephan) to the patch or provide a secret url so people can check

**Reply by akerb00m** [reply]
you are invited : )
i invited you to this one to, https://cables.gl/edit/2D7eks its a remake of the one that broke maybe that helps ?

**Reply by akerb00m** [reply]
sorry i don't remember the error message, it kind off went crazy and i didn't pay enough attention to the error message ...

**Reply by steam0r** [reply]
this does not crash (for me)

**Reply by akerb00m** [reply]
do you mean this one ? 
(broken) - https://cables.gl/edit/WYrvas  
it doesn't crash when you open it, its just broken...

It functioned just like this one 
(working) - https://cables.gl/edit/2D7eks 
but all you can still do is move the floor grid.
is it not like that for you ?

**Reply by steam0r** [reply]
you worked around this somehow, as said on discord, right? in that "broken" patch your orbitcontrols do not "orbit" anything (just for the record :))

**Reply by akerb00m** [reply]
In the working patch, build the whole thing new, from scratch. so not really a work around ; )
i connected the orbit controls to the MESH trigger in the broken patch again but it is still broken.

**Reply by steam0r** [reply]
can you share how your rework is different?

**Reply by akerb00m** [reply]
The only thing that is different is the amount of GLB files attached under the 'MESH' trigger (the reworked patch has more) 
The Broken Patch broke when I tried to add a phong light and a shadow operator, i did not add those operators again in the reworked patch
